File: JavaScript/e009.js
let c, answer;
for (let b = 4; b < 1000; b++) {
    for (let a = 1; a <= b; a++) {
        c = Math.sqrt(a ** 2 + b ** 2);
        if (c === Math.floor(c) && a + b + c === 1000) {
            answer = a * b * Math.floor(c);
            break;
        }
    }
}
console.log(answer);

File: e009.py
"""
Problem 9: Special Pythagorean Triplet
There exists exactly one Pythagorean triplet for which a+b+c = 1000.
Find the product abc.
"""
from math import sqrt

for b in range(4, 1000):
    for a in range(1, b + 1):
        c = sqrt(a ** 2 + b ** 2)
        if c == int(c) and a + b + c == 1000:
            print(a * b * int(c))
            break
